### ISSUE TITLE
Metrics format has changed in v1.9

### ISSUE THREAD

**Issue by kubotat** [fluent/fluent-bit#5220]
## Bug Report

**Describe the bug**

I've tested node_exporter_metrics and fluentbit_metrics input plugins and it seems that metrics format of v1.9 has changed from v1.8. In terms v1.8, each metrics has epoch time but it does not exist in v1.9. 
Is the change, epoch time does not exist in v1.9, expected behavior? If this is expected behavior, could you please inform me of the reason/back ground why this change was made?

- node_exporter_metrics output of Fluent Bit v1.8.12
```
# HELP node_cpu_seconds_total Seconds the CPUs spent in each mode.
# TYPE node_cpu_seconds_total counter
node_cpu_seconds_total{cpu="0",mode="idle"} 15550319.85 1648704606281
node_cpu_seconds_total{cpu="0",mode="iowait"} 414.05000000000001 1648704606281
node_cpu_seconds_total{cpu="0",mode="irq"} 188411.42999999999 1648704606281
node_cpu_seconds_total{cpu="0",mode="nice"} 2494.3499999999999 1648704606281
node_cpu_seconds_total{cpu="0",mode="softirq"} 4968.3400000000001 1648704606281
node_cpu_seconds_total{cpu="0",mode="steal"} 46432.690000000002 1648704606281
node_cpu_seconds_total{cpu="0",mode="system"} 314173.71000000002 1648704606281
node_cpu_seconds_total{cpu="0",mode="user"} 166170.23000000001 1648704606281
node_cpu_seconds_total{cpu="1",mode="idle"} 15593238.82 1648704606281
node_cpu_seconds_total{cpu="1",mode="iowait"} 1823.0799999999999 1648704606281
node_cpu_seconds_total{cpu="1",mode="irq"} 175412.66 1648704606281
node_cpu_seconds_total{cpu="1",mode="nice"} 2580.3299999999999 1648704606281
node_cpu_seconds_total{cpu="1",mode="softirq"} 4583.8699999999999 1648704606281
node_cpu_seconds_total{cpu="1",mode="steal"} 45426.239999999998 1648704606281
node_cpu_seconds_total{cpu="1",mode="system"} 300571.13 1648704606281
node_cpu_seconds_total{cpu="1",mode="user"} 157451.53 1648704606281
```
- node_exporter_metrics output of Fluent Bit v1.9.0
```
# HELP node_cpu_seconds_total Seconds the CPUs spent in each mode.
# TYPE node_cpu_seconds_total counter
node_cpu_seconds_total{cpu="0",mode="idle"} 15550291.52
node_cpu_seconds_total{cpu="0",mode="iowait"} 414.05000000000001
node_cpu_seconds_total{cpu="0",mode="irq"} 188410.94
node_cpu_seconds_total{cpu="0",mode="nice"} 2494.3499999999999
node_cpu_seconds_total{cpu="0",mode="softirq"} 4968.3199999999997
node_cpu_seconds_total{cpu="0",mode="steal"} 46432.550000000003
node_cpu_seconds_total{cpu="0",mode="system"} 314172.83000000002
node_cpu_seconds_total{cpu="0",mode="user"} 166169.84
node_cpu_seconds_total{cpu="1",mode="idle"} 15593210.630000001
node_cpu_seconds_total{cpu="1",mode="iowait"} 1823.0799999999999
node_cpu_seconds_total{cpu="1",mode="irq"} 175412.10000000001
node_cpu_seconds_total{cpu="1",mode="nice"} 2580.3299999999999
node_cpu_seconds_total{cpu="1",mode="softirq"} 4583.8599999999997
node_cpu_seconds_total{cpu="1",mode="steal"} 45426.139999999999
node_cpu_seconds_total{cpu="1",mode="system"} 300570.19
node_cpu_seconds_total{cpu="1",mode="user"} 157450.98000000001
```
**To Reproduce**
- Run Fluent Bit with the following configuration. You can see the difference in metrics format between v1.8 and v1.9.
```
[SERVICE]
    Flush      5
    Daemon     off
    Log_Level  info

[INPUT]
    name         node_exporter_metrics
    tag             node_metrics
    scrape_interval 2

#[INPUT]
#   Name fluentbit_metrics
#   Tag fluentbit.metrics
#    scrape_on_start true
#    scrape_interval 30

[OUTPUT]
    Name          stdout
    Match         *
```

**Expected behavior**

**Screenshots**

**Your Environment**

* Version used: v1.8.12 and v1.9.0
* Configuration: 
```
[SERVICE]
    Flush      5
    Daemon     off
    Log_Level  info

[INPUT]
    name         node_exporter_metrics
    tag             node_metrics
    scrape_interval 2

#[INPUT]
#   Name fluentbit_metrics
#   Tag fluentbit.metrics
#    scrape_on_start true
#    scrape_interval 30

[OUTPUT]
    Name          stdout
    Match         *
```
* Environment name and version (e.g. Kubernetes? What version?): Onprem
* Server type and version: 
* Operating System and version: RHEL/CentOS8
* Filters and plugins: node_exporter_metrics and fluentbit_metrics

**Additional context**

**Comment by nokute78** [comment]
I think https://github.com/fluent/fluent-bit/pull/4819 relates this issue.
Commit is https://github.com/fluent/fluent-bit/commit/0eda9bde469f3e9e7c06c05662961275a32b3b85

**Comment by hisa-tanaka** [comment]
@nokute78 thank you. With that being said in #4819, adding / removing timestamp is configurable, and 'add_timestamp' can add timestamps to metrics to v.1.9, correct?

node_cpu_seconds_total{cpu="1",mode="user"} 157451.53 *1648704606281*

To be sure, will you please let me know which part of the config shall I add the "add_timestamp"? 

cc @kubotat

**Comment by edsiper** [comment]
In early v1.8.x series, out_prometheus exporter set always the known metric timestamp. By default (in the industry) timestamps are set by the client who's collecting the metric. In the latter 1.8.x we shipped a new default where timestamp are not included, but, a new option exists to bring them back.

Internal metrics format has not changed, just what we exposed in prometheus exporter with a new option. 

@hisa-tanaka this change only applies to prometheus exporter plugin, not forward.
